package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "components.underscores.me",
+  "version": "1.0.0",
+  "description": "components.underscores.me website",
+  "main": "index.js",
+  "scripts": {
+    "build": "npm run gulp build",
+    "gulp": "gulp",
+    "start": "npm run gulp watch",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Automattic/components.underscores.me.git"
+  },
+  "keywords": [
+    "wordpress",
+    "themes",
+    "theme"
+  ],
+  "author": "Automattic",
+  "license": "GPL-2.0",
```

Comment:
I assumed this was GPL 2.0 as most (all?) of Automattic's open source code seems to be, but I wasn't sure hence #96